index.js

```diff
@@ -2,19 +2,516 @@ const readline = require('readline');
+		console.log(`You dropped a ${droppableItems.item}`);
+	} else {
+		console.log('I cant...');
+	}
+};
+
+//--------Global look around function to be called on in any room --------//
+let lookAround = function () {
+	if (roomInv.length !== 0) {
+		for (let roomItems of roomInv) {
+			collectedItem = roomItems.item;
+		}
+		console.log('\n' + roomDescription);
+		for (let roomItems of roomInv) {
+			console.log('you see a ' + roomItems.item);
+		}
+	} else if (roomInv.length === 0) {
+		console.log(roomDescription);
+		console.log("There's nothing here..");
+	}
+};
+
+//-------global check inventory function that will take the items in the player inventory array and display there item name.
+let checkInventory = function () {
+	if (playerInventory.length > 0) {
+		for (let items of playerInventory) {
+			droppableItems = items;
+			console.log(droppableItems.item + ': ' + droppableItems.description);
+		}
+	} else if (playerInventory.length <= 0) {
+		console.log('Empty');
+	}
+};
+
+//-----making a list of room descriptions to call on inside global look around functions
+roomDescription = undefined;
+let lakeRoomDescription =
+	'You are in a clearing with a small lake.. the water looks questionable, but you are so terribly thirsty...\n';
+let riverRoomDescription = 'There is massive river, obstructing all movement except to the north\n';
+let meadowRoomDescription =
+	'A foreboding mountain lies to the East and West. A path cuts through the woods to the north, and there is the sound of a river to the south\n';
+let puzzleRoomDescription =
+	'You come to a forrest. It is shady aside from some slivers of light breaking through the leaves.';
+let deadEndRoomDescription = 'You now stand at the edge of a giant cliff.';
+let wellRoomDescription = 'You are in a field with a well.';
+let lockedRoomDescription =
+	'You now come to a gate, guarded by a troll. The troll will not let you pass, unless he is paid off.';
+let finalRoomDescription =
+	'You are in a cave. There is a portal to the south, which appears to lead home, but not much else.';
 
+class Item {
+	constructor(item, description, collectable) {
+		this.item = item;
+		this.description = description;
+		this.collectable = collectable || false;
+	}
+}
+
+let stick = new Item('stick', "Just a plain ol' stick nothing interesting", true);
+
+let bucket = new Item('bucket', 'Could be used to get water from a well', true);
+
+let rock = new Item('rock', 'this rock is too heavy to lift', false);
+
+let goldCoin = new Item('gold coin', 'could be used to pay someone off', true);
+
+let immovableItemsTable = ['rock', 'gold coin'];
+let acceptableNorthCommands = ['n', 'N', 'North', 'NORTH'];
+let acceptableSouthCommands = ['s', 'S', 'South', 'SOUTH'];
+let acceptableEastCommands = ['e', 'E', 'east', 'EAST'];
+let acceptableWestCommands = ['w', 'W', 'West', 'WEST'];
+let acceptableTakeCommands = ['take', 'Take', 'TAKE'];
+let acceptableLookCommands = ['scan', 'look around', 'LOOK', 'LOOK AROUND', 'look'];
```

Comment:
Good use of class constructors here. Also creating arrays to check for commands and items with specific properties is useful.